src/components/MovieDetailsHeader/MovieDetailsHeader.js

```diff
@@ -0,0 +1,60 @@
+import React, {useEffect, useState} from "react";
+import Logo from "../Logo/Logo.js";
+import useComponentDidMount from "../../utils/hooks.js";
+import "./style.scss";
+
+const fetchMovieDetails = () => ({
+  title: "Pulp Fiction",
+  releaseDate: 1994,
+  rating: 4.3,
+  runtime: "120 min",
+  imageUrl: 'https://thumbs.dfs.ivi.ru/storage2/contents/3/e/506516e4dc81c88a4de176bde11374.jpg',
+  description: 'Pulp Fiction is a 1994 American neo-noir black comedy crime film written and directed by Quentin Tarantino, who conceived it with Roger Avary. Starring John Travolta, Samuel L. Jackson, Bruce Willis, Tim Roth, Ving Rhames, and Uma Thurman, it tells several stories of criminal Los Angele',
+  id: 1,
+});
+
+export default function MovieDetailsHeader() {
+  const [movie, setMovie] = useState({});
+
+  useComponentDidMount(() => {
```

Comment:
Just a proposal, you could move your state also inside this custom hook and just return a movie from the hook